lib/pure/times.nim

```diff
@@ -2721,3 +2787,51 @@ proc getGMTime*(time: Time): DateTime
   ## expressed in Coordinated Universal Time (UTC).
   # Deprecated since v0.18.0
   time.utc
+
+proc `nanosecond=`*(dt: var DateTime, value: NanosecondRange) {.deprecated.} =
+  # Deprecated since v1.4.0
```

Comment:
* this comment should be inside `{.deprecated: "since v1.3.1" .}` so it appears both in deprecation messages as well as nim doc
* it's 1.3.1, not 1.4.0 (just like all the since statements that use current nim version, regardless of whether it's a release or not; this makes nim consistent both in devel and in releases)

Reply:
Fixed